DESCRIPTION

```diff
@@ -21,10 +21,7 @@ Imports:
 Suggests:
     highr,
     Cairo,
-    testthat,
-    data.table,
-    dplyr,
-    htmlwidgets
+    testthat
 Enhances: data.table, dplyr, htmlwidgets
```

Comment:
Then what is the difference to data.table? Both provide a data.frame like class and are tested in to be compliant in the "only display start and end of a frame"...